pyproject.toml

```diff
@@ -50,10 +46,12 @@ dependencies = [
 Homepage = "https://github.com/descendant-ai/functime"
 
 [project.optional-dependencies]
+ann = ["pylance"]
 cat = ["catboost"]
 xgb = ["xgboost"]
 lgb = ["lightgbm"]
 llm = ["openai", "tabulate", "tenacity", "tiktoken"]
+plot = ["plotly", "kaleido"]
```

Comment:
Perhaps you should write `"kaleido==0.2.1"`?

Reply:
👍